# This patch removes tests from the Makefile.
# This is because tests require either internet access or root permissions, things not allowed during the build.

--- onedriver-0.13.0/Makefile
+++ onedriver-0.13.0-patched/Makefile
@@ -105,23 +105,6 @@
 	curl ftp://ftp.ensemblgenomes.org/pub/metazoa/release-42/fasta/drosophila_melanogaster/dna/Drosophila_melanogaster.BDGP6.22.dna.chromosome.X.fa.gz | zcat > $@
 
 
-# For offline tests, the test binary is built online, then network access is
-# disabled and tests are run. sudo is required - otherwise we don't have
-# permission to mount the fuse filesystem.
-test: onedriver onedriver-launcher dmel.fa
-	rm -f *.race* fusefs_tests.log
-	CGO_ENABLED=0 gotest -v -parallel=8 -count=1 $(shell go list ./ui/... | grep -v offline)
-	GORACE="log_path=fusefs_tests.race strip_path_prefix=1" \
-		gotest -race -v -parallel=8 -count=1 ./cmd/...
-	GORACE="log_path=fusefs_tests.race strip_path_prefix=1" \
-		gotest -race -v -parallel=8 -count=1 ./fs/graph/...
-	GORACE="log_path=fusefs_tests.race strip_path_prefix=1" \
-		gotest -race -v -parallel=8 -count=1 ./fs
-	go test -c ./fs/offline
-	@echo "sudo is required to run tests of offline functionality:"
-	sudo unshare -n -S $(TEST_UID) -G $(TEST_GID) ./offline.test -test.v -test.parallel=8 -test.count=1
-
-
 # will literally purge everything: all built artifacts, all logs, all tests,
 # all files tests depend on, all auth tokens... EVERYTHING
 clean:
